extensions/VisualEditor/wikia/modules/ve/ve.track.js

```diff
@@ -199,6 +199,11 @@ require( ['wikia.tracker'], function ( tracker ) {
 		funnelEvent = data.label + '/' + data.action;
 
 		if ( funnelEvents.indexOf( funnelEvent ) !== -1 ) {
+			if (ve.init.wikia.getToolbarABTestVariantNumber() === 1 && funnelEvents === 'edit-page-ready/impression') {
+				// Extra impression event when toolbar A/B test is active
```

Comment:
in ve everything is tracked as an event, and subset of everything is tracked as a fake page view. after this change we will have fake page view which does not have a corresponding event. just making sure that's what you want. (usually we use events for tracking particular features metrics / success)